internal/usage/usage.go

```diff
@@ -264,6 +264,7 @@ const (
 	ContainerRegion                 = "Atlas region where the container resides."
 	ContainerRegions                = "List of Atlas regions where the container resides."
 	ProjectOwnerID                  = "Unique 24-hexadecimal digit string that identifies the Atlas user account to be granted the Project Owner role on the specified project."
+	GovCloudRegionsOnly             = "Make the project gov region only."
```

Comment:
this should probably clarify only available for Atlas Gov

Reply:
Added more detail. Is this correct: if you use this flag the project uses GovCloud region, and if you don't use it then the project uses Standard region?
```suggestion
	GovCloudRegionsOnly             = "Only for AtlasGov organizations. If specified, designates that the project uses the AWS GovCloud region only. If unspecified, the project uses the AWS Standard region only. You can't deploy clusters across AWS GovCloud and AWS Standard regions in the same project."
```